mountpoint-s3-fs/src/manifest/csv_reader.rs

```diff
@@ -35,25 +72,34 @@ impl<R: Read> Iterator for CsvReader<R> {
 ///
 /// Note that data in this struct is not yet validated.
 /// Specifically [partial_key] may contain forbidden symbols and be rejected later.
-#[derive(Deserialize)]
-struct CsvEntry {
+#[derive(Deserialize, Serialize)]
+pub struct CsvEntry {
     /// Partial key of the S3 object.
     ///
     /// Must not contain S3 prefix, when the prefix is mounted.
     /// May hold a directory marker (e.g. "dir1/dir2/"), which will be skipped
     /// and won't affect the file system structure.
-    partial_key: String,
+    pub partial_key: String,
     /// Etag of the S3 object.
-    etag: String,
+    pub etag: String,
     /// Size of the S3 object.
-    size: usize,
+    pub size: usize,
```

Comment:
This should be `u64` to remove some of the ambiguity on the checksum calculation.